10.9-libcxx/stable/main/finkinfo/devel/meson-py.info

```diff
@@ -0,0 +1,103 @@
+# -*- coding: ascii; tab-width: 4 -*-
+Info2: <<
+Package: meson-py%type_pkg[python]
+Version: 0.62.2
+Revision: 1
+Type: python (3.7 3.8 3.9 3.10)
+Depends: python%type_pkg[python], ninja (>= 1.9-1)
+BuildDepends: fink (>= 0.24.12), setuptools-tng-py%type_pkg[python]
```

Comment:
nit: no need for ```fink (>= 0.24.12)``` BDep

Reply:
No idea how I came up with 0.24.12. But I thought `Info2` still required 0.20.0; validation passes without any fink [B]Dep... – maybe assuming there are no more ancient versions in existence!